core/src/main/scala/com/rallyhealth/vapors/factfilter/data/Fact.scala

```diff
@@ -1,12 +1,25 @@
 package com.rallyhealth.vapors.factfilter.data
 
-import com.rallyhealth.vapors.core.data.{DataPath, NamedLens}
+import com.rallyhealth.vapors.core.data.{DataPath, FieldLens, NamedLens}
 
 sealed abstract class Fact {
   type Value
 
   val typeInfo: FactType[Value]
   val value: Value
+
+  override def toString: String = {
+    // TODO: JSON format for value? Show for value?
```

Comment:
I want to separate the JSON encoding from the fact, but I need to figure out how to annotate untyped facts for proper JSON serialization for debugging.

Reply:
I'll do this in a follow-up PR